server/matchmaker/matchmaker_queue.py

```diff
@@ -278,6 +278,10 @@ def to_dict(self):
         """
         Return a fuzzy representation of the searches currently in the queue
         """
+        if self.team_size == 1:
+            ratings = {round(search.ratings[0].mean / 25) * 25 for search in self._queue.keys()}
+        else:
+            ratings = {round(search.average_rating / 25) * 25 for search in self._queue.keys()}
```

Comment:
Shouldn't `average_rating` always be correct even if team size is 1? Average of x is just x.
```suggestion
        ratings = {round(search.average_rating / 25) * 25 for search in self._queue.keys()}
```

Reply:
average rating uses displayed rating, while the 1v1 queue uses mean rating for matching. Maybe a comment should explain this?